plugins/modem-manager/fu-mm-device.c

```diff
@@ -752,7 +912,9 @@ fu_mm_device_attach (FuDevice *device, GError **error)
 
 	/* wait for re-probing after uninhibiting */
 	fu_device_set_remove_delay (device, FU_MM_DEVICE_REMOVE_DELAY_REPROBE);
-	fu_device_add_flag (device, FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG);
+	if (!(self->update_methods & MM_MODEM_FIRMWARE_UPDATE_METHOD_MBIM_QDU))
```

Comment:
I think `(self->update_methods & MM_MODEM_FIRMWARE_UPDATE_METHOD_MBIM_QDU) == 0` would be clearer here

Reply:
Ok, I will change it, thanks.